lib/core/main.js

```diff
@@ -31,7 +31,8 @@ const FormData = require('form-data');
  * @param {number} [options.page] - Page number.
  * @param {boolean} [options.parse_ads] - Weather or not to parse ads.
  * @param {boolean} [options.use_mobile_ua] - Weather or not to use a mobile user agent.
- * @param {object} [options.additional_params] - parameters that will be passed to Google
+ * @param {object} [options.additional_params] - Additional parameters that will be passed to Google.
```

Comment:
refactoring: aligned with [other function](https://github.com/LuanRT/google-this/pull/38/files#diff-c8065054c65fe77edfce585a229adcd8d39fbcd0258aa3f4791b4063aa1fda6eR146)